src/pudl/transform/eia860.py

```diff
@@ -161,6 +161,19 @@ def clean_ownership_eia860(raw_ownership_eia860: pd.DataFrame) -> pd.DataFrame:
     own_df["owner_country"] = own_df["owner_state"].map(state_to_country)
     own_df.loc[own_df.owner_state == "CN", "owner_state"] = pd.NA
 
+    # Spot fix NA generator_id. Might want to change this once we have the official 2022
+    # data not just early release.
+    constraints = (own_df["plant_id_eia"] == 62844) & (
+        own_df["report_date"].dt.year == 2022
+    )
+
+    if len(own_df[constraints]) > 1:
+        raise AssertionError("Too many records getting spot fixed.")
+    if own_df[constraints].generator_id.notna().all():
+        raise AssertionError("Generator ID filled in, you can remove this hack!")
+    else:
+        own_df.loc[constraints, "generator_id"] = "1"
+
```

Comment:
i think if you added `elif 2022 not in own_df.report_date.dt.year.unique(): pass` into this mix this would enable a non-2022 run.

perhaps you could just add it into the if own[constraints] line?

Reply:
Ah right, I figured it wasn't necessary because we are adding 2022 in, but there will be a time in the future when 2022 isn't in the fast tests.